IBM/carbon-addons-iot-react
# [DateTimePicker] Date picker issues

Closes #1183 closes #1179 

**Summary**

This PR addresses both issues above as well as a bug that exist when you click on custom option and switch back to presests. If you click apply it will apply the preset value even if you had not clicked one.

Another bug is with translations where there could be a mismatch of the translation value and the value that is populated in the select dropdown. 


**Change List (commits, features, bugs, etc)**

- Changed default story to use `relatives` prop
- Change the without relative option story to have preset as default value (#1179)
- added presets to base value instead of being null
- when resetting the relative value, setting it to the first item in array instead of blank string
- checked if the current selected `relativeWhen` matches a translation string and either show the appropriate string or the value of the select. 

**Acceptance Test (how to verify the PR)**

- Go to DateTimePicker story. 
- Go to last story. Try to go to custom range and see that calendar is rendered and not the relative option. 


For the rest just go through stories and click values and then change to other type (preset/custom)
